base/applications/utilman/umandlg/process.c

```diff
@@ -84,7 +84,7 @@ BOOL IsProcessRunning(IN LPCWSTR lpszProcessName)
     hProcess = OpenProcess(SYNCHRONIZE, FALSE, dwProcessID);
     if (!hProcess)
     {
-        DPRINT("IsProcessRunning(): Failed to open the process! (Error: %lu)", GetLastError());
+        DPRINT("IsProcessRunning(): Failed to open the process! (Error: %lu)\n", GetLastError());
```

Comment:
```suggestion
        DPRINT("Failed to open the process! (Error: %lu)\n", GetLastError());
```
DPRINT logs the function name itself already.

Reply:
No, it just logs the source file path and line number.

Reply:
Really?

Reply:
You are right. It doesn't. My fault. Resolving this request.